main.py

```diff
@@ -117,18 +38,27 @@ def main() -> int:
         params=model.parameters()
     )
 
+    experiment = Experiment(
+        api_key=os.environ['COMET_ML_API_KEY'],
```

Comment:
a to nie jest tak, że jak nie podamy api_key to comet sam z siebie poszuka w zmiannych środowiskowych?

Reply:
nie mam pojecia, tak jest jawnie xd